pkg/trace/config/config.go

```diff
@@ -373,6 +385,9 @@ type AgentConfig struct {
 	// AppSec contains AppSec configuration.
 	AppSec AppSecConfig
 
+	// DebuggerProxy contains the settings for the Live Debugger proxy.
```

Comment:
```suggestion
	// EVPIntakeProxy ...
```
Whoopsy copy 🍝 accident